### ISSUE TITLE
[BUG] fix `releases.yml` workflow version number

### ISSUE THREAD

**Issue by Pradumnasaraf** [Pradumnasaraf/LinkFree-CLI#18]
### Description

Update the 2 actions to the latest version

```diff
- uses: actions/checkout@v2
+ uses: actions/checkout@v3
```

```diff
- uses: TriPSs/conventional-changelog-action@v3.7.1
+ uses: TriPSs/conventional-changelog-action@v3
```

File path - https://github.com/Pradumnasaraf/LinkFree-CLI/blob/main/.github/workflows/releases.yml

**Comment by Shipkush** [comment]
hello!....please assign this issue to me.

**Comment by NikhilRaikwar** [comment]
Hello @Pradumnasaraf Bro, I'm interested in working on this issue. Would you be willing to assign it to me? I'm excited to help out and I'll do my best to resolve this issue as quickly as possible. Please let me know if there's anything I can do to help. Thanks!

**Comment by Pradumnasaraf** [comment]
> Hello @Pradumnasaraf Bro, I'm interested in working on this issue. Would you be willing to assign it to me? I'm excited to help out and I'll do my best to resolve this issue as quickly as possible. Please let me know if there's anything I can do to help. Thanks!
> 

Go ahead. Thank you.

**Comment by Pradumnasaraf** [comment]
Fixed #22
